src/test/resources/application-integration-test.yml

```diff
@@ -22,6 +22,9 @@ spring:
         format_sql : false
         show_sql : false
 
+  codec:
```

Comment:
We need this new property in the integration test spring context (`spring.codec.max-in-memory-size`) to allow the `WebTestClient` that is used in the integration tests to handle a large response body
(we have an integration test that generates and downloads the explainer doc - now we have the real explainer document template PDFs, the resultant http response is larger (almost 1MB !!)))

Reply:
I would agree that your approach is more explicit. The property I have used is not exactly self-explanatory !!
I'll change it to use your approach 👍

Reply:
Done